src/NServiceBus.Gateway.AcceptanceTests/When_using_legacy_inmemory_persistence.cs

```diff
@@ -31,7 +31,8 @@ public EndpointWithLegacyConfiguration()
                     c.UsePersistence<InMemoryPersistence, StorageType.GatewayDeduplication>();
                     
                     // legacy configuration API (not passing a storage as parameter):
-                    c.Gateway();
+                    var gatewayConfiguration = c.Gateway();
+                    gatewayConfiguration.AddReceiveChannel("http://localhost:25999/SiteA/");
```

Comment:
Why was this API not required in the test before?

Reply:
this is a new test and the test failed as the default receive channel url doesn't seem to be configured on the build-agent. Not really sure how it's done, but the specified URL is used by multiple tests and seems to be allowed on the agent.